src/dashboards/Compact/index.tsx

```diff
@@ -501,7 +535,7 @@ const EnturDashboard = (): JSX.Element | null => {
                                 />
                             </div>
                         ))}
-                        {bikeRentalStations && anyBikeRentalStations ? (
+                        {bikeRentalStations && bikeRentalStations.length > 0 ? (
```

Comment:
mulig å bruke bicyclestopplaces.loading og/eller error her for å bestemme om man skal vise den tilen eller ikke?

Reply:
det kommer i mobilitytile